src/views/home/home.view.jsx

```diff
@@ -150,9 +169,16 @@ function Home ({
     copyToClipboard(hermezEthereumAddress)
     onOpenSnackbar('The Hermez address has been copied to the clipboard!')
   }
-
+  
   return wallet && (
     <div className={classes.root}>
+      {
+        estimatedRewardTask.status === 'successful' && earnedRewardTask.status === 'successful' 
+          ? <Sidenav onClose={onClose}>
```

Comment:
I would render the `Sidenav` component inside the `AirdropPanel` component here as well.

Reply:
Same as above.